.docker/src-cms/etc/nginx/partials/drupal.conf.tmpl

```diff
@@ -79,6 +79,18 @@
     }
 
     location / {
+        # Decode the URI for proper matching
+        set $decoded_uri $uri;
+        if ($decoded_uri ~* ^(.+?)(%C2%A0|\xA0|%E2%80%89|\xE2\x80\x89|%E2%80%83|\xE2\x80\x83)+$) {
```

Comment:
I would like a comment here -- which characters are these?

Reply:
Ah good idea, this is now done 👍